src/org/omegat/filters3/xml/resx/ResXFilter.java

```diff
@@ -120,7 +121,7 @@ public void tagEnd(String path) {
         if ("/root/data/comment".equals(path)) {
             comment = text;
         } else if ("/root/data".equals(path)) {
-            if (entryParseCallback != null) {
+            if (entryParseCallback != null && entryText !=null) {
```

Comment:
```suggestion
            if (entryParseCallback != null && entryText != null) {
```

to be better code style.